src/NLog/LayoutRenderers/Wrappers/Rot13LayoutRendererWrapper.cs

```diff
@@ -43,7 +43,6 @@ namespace NLog.LayoutRenderers.Wrappers
     /// See <a href="http://en.wikipedia.org/wiki/ROT13">http://en.wikipedia.org/wiki/ROT13</a>.
     /// </remarks>
     [LayoutRenderer("rot13")]
-    [AppDomainFixedOutput]
```

Comment:
why is this not needed anymore? I don't see it.

Reply:
Thought the attribute meant that the output of the layoutrenderer would be completely static in the life time of the AppDomain (Ex like for newline-layoutrenderer). Cannot see how the same goes for Rot13. Anyway not important for me, so I can re-introduce it, if you want.

Reply:
I was thinking it was same output when same input, so independent of other renderers. Need to check that.

Reply:
Checked this, I think ` [AppDomainFixedOutput]` is correct here.  From the code:

a layout(renderer) could be converted to a literal when: 
- The layout and inner layouts are SimpleLayouts, or `[AppDomainFixedOutput]`

So if rot13 has a `SimpleLayout`, it will always have the same output.

Reply:
Yes you are right. Actually means that all wrapper layouts should have AppDomainFixedOutput